src/platforms/react-native/troubleshooting.mdx

```diff
@@ -160,3 +160,8 @@ dyld: Library not loaded: /usr/local/opt/icu4c/lib/libicui18n.62.dylib
 ```
 
 In this case, type `ln -s $(which node) /usr/local/bin/node`, which tells your env to use your node binary.
+
+## NavigationContainer reset the ref each render
+
+If you are using `React Navigation 5` and you've experienced not receiving transactions anymore, it might be because of this [Bug](https://github.com/react-navigation/react-navigation/issues/9213), upgrading to `React Navigation 6` should solve the issue, if not, file an issue in our [repo](https://github.com/getsentry/sentry-react-native).
```

Comment:
```suggestion
If you're using `React Navigation 5` and you're not receiving transactions anymore, it might be because of [this bug](https://github.com/react-navigation/react-navigation/issues/9213). Upgrading to `React Navigation 6` should solve the issue, but if not, file an issue in our [repo](https://github.com/getsentry/sentry-react-native).
```